bulk-model-sync-gradle-test/build.gradle.kts

```diff
@@ -14,13 +14,10 @@
  * limitations under the License.
  */
 
-import org.modelix.model.server.Main
-
 buildscript {
     val modelixCoreVersion: String = file("../version.txt").readText()
     dependencies {
         classpath("org.modelix:model-server:$modelixCoreVersion")
-        classpath("org.modelix:graph-lang-api:$modelixCoreVersion")
     }
 }
```

Comment:
Now that we don't read the default port from the model-server anymore, we also no longer need the buildscript dependencies.